internal/algod/mac/mac.go

```diff
@@ -61,7 +64,7 @@ func Install() error {
 
 	// Create and load the launchd service
 	// TODO: find a clever way to avoid this or make sudo persist for the second call
```

Comment:
re: comment; sudo would persist for a certain time interval on most shells/systems, no?